lib/block/block.js

```diff
@@ -62,27 +51,35 @@ Block._from = function _from(arg) {
 };
 
 /**
- * @param {String|Object} - A JSON string or object
+ * @param {String} - A JSON string
  * @returns {Object} - An object representing block data
  * @private
  */
 Block._fromJSON = function _fromJSON(data) {
```

Comment:
I'd say we should make this a part of another PR that we can go through add those methods where necessary, so that there there are round trips for `fromJSON/toJSON` and `fromObject/toObject`. However if either of those handle both a string and object, we only need one pair.